ractor/src/macros/macro.rs

```diff
@@ -237,3 +237,6 @@ macro_rules! forward {
         }
     }};
 }
+
+#[cfg(test)]
+mod tests;
```

Comment:
the file move isn't necessary here, you can keep it at /src/macros.rs and tests will still resolve to /src/macros/test.rs